lib/npm-util.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const execa = require('execa');
+const pTimeout = require('p-timeout');
+
+exports.connectToNpm = () => pTimeout(
+	(async () => {
+		try {
+			await execa('npm', ['ping']);
+			return true;
+		} catch (_) {
+			throw new Error('Connection to npm registry failed');
+		}
+	})(),
+	15000,
+	'Connection to npm registry timed out'
+);
+
+exports.npmUsername = async () => {
+	try {
+		return await execa.stdout('npm', ['whoami']);
+	} catch (error) {
+		return error;
+	}
+};
+
+exports.npmCollaborators = async pkgName => {
+	try {
+		return await execa.stdout('npm', ['access', 'ls-collaborators', pkgName]);
+	} catch (error) {
+		// Ignore non-existing package error
+		if (error.stderr.includes('code E404')) {
+			return '404';
```

Comment:
Returning `false` when the package doesn't exist on npm, the collaborators in case they were found successfully and throwing any error that comes up sounds like a good option to me.